tests/benchdnn/inputs/conv/test_conv_depthwise

```diff
@@ -16,3 +16,7 @@
 --batch=set_conv_dw
 --batch=shapes_dw_minibatch_2d-spatial
 --batch=shapes_dw_minibatch_channel_2d-spatial
+
+--reset --dt=bf16 --bia-dt=bf16
+--mb=2
+--dir=FWD_D,FWD_I --batch=shapes_mobilenet_dw
```

Comment:
I'm a bit on the fence about these new testcases, as these will be executed on all supported platforms in perpetuity.